t/local/head.t

```diff
@@ -0,0 +1,31 @@
+use warnings;
+use strict;
+use Test::More tests => 8;
```

Comment:
I think you're following the pattern of other tests, but I don't think we need to do that. Using `done_testing()` should be good enough.

Reply:
I had that first, then undid it because of the old tests. Probably about time we clean those up.